src/components/Accordion.js

```diff
@@ -0,0 +1,93 @@
+const menuOpen = "collapse show";
+const buttonOpenAll = "Open All";
+const buttonCloseAll = "Close All";
+
+if (!String.prototype.includes) {
+  String.prototype.includes = function(search, start) {
+    "use strict";
+    if (typeof start !== "number") {
+      start = 0;
+    }
+
+    if (start + search.length > this.length) {
+      return false;
+    } else {
+      return this.indexOf(search, start) !== -1;
+    }
+  };
+}
+
+document.addEventListener(
+  "click",
+  clickEvent => {
+    const { target } = clickEvent;
+
+    if (target && target.className === "dg_allitems") {
+      AllMenuItemsAction(target);
+    } else if (target) {
+      MenuAction(target);
+    } else {
+      return;
+    }
+  },
+  false
+);
+
+function MenuAction(element) {
+  var mainDiv = element.parentElement.parentElement;
```

Comment:
this is dangerous, if any of the html structure changes.

I would suggest using [closest](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest). I think this may need to be polyfilled, but I had to use this so I can get it to you.

Reply:
I get what youre sayig here but we run the risk of any of this breaking if they decide to change the html too drastically. The key to this is that the html and css shoudlnt change once its approved. Using css classes to grab these elements can run into the exact issue if we end up using a new css class name as well. I'll change this per your suggestion but the design team can break this a 1000 ways by modifying html or css. Plus we are now knowingly introducing a break in ie11 and having to polyfill. Seems like a drastic change to allow the design team to keep making changes to already approved html structure.

Reply:
This isn't necessarily anything the design may do, this might be something we overlook. This is something I have learned the hard way and makes our code much more resilient. This technique comes from years of jquery experience, in which closest is built in.

If we need to introduce jQuery, because we aren't comfortable poly-filling, i am okay with that discussion, I am used to adding polyfills as of working with IE11, but I didn't run that by you guys so I can understand the potential frustration.

let me know if we want to discuss reintroducing jquery.

Reply:
No need to add jquery back. I'll change to closest and already have a polyfill for it but these two solutions seem to be interchangeable and this is more a code refactor than anything else which adds time to the process. But I will do it.

Reply:
This is done.